jsplot/src/test/java/tech/tablesaw/components/LayoutTest.java

```diff
@@ -55,58 +58,62 @@ public void testAutosize() {
     {
       Layout layout = Layout.builder().autosize(true).build();
       assertEquals(
-          "var layout = {\n"
-              + //
-              "    autosize: true,\n"
-              + //
-              "\n\n"
-              + //
-              "};\n",
+          "var layout = {"
+              + lineSeparator()
```

Comment:
this one should be `LINE_END` to match the others and then you no longer need the static `import`